resources/views/individuals/edit/about-you.blade.php

```diff
@@ -83,14 +83,14 @@
 
             <fieldset>
                 <legend>{{ __('Social media links') }}</legend>
-                <p class="field__hint">
+                <x-hearth-hint for="social_links">
                     {{ __('Website links must be in the format “https://example.com”, or “example.com”.') }}
-                </p>
+                </x-hearth-hint>
                 @foreach (['linked_in', 'twitter', 'instagram', 'facebook'] as $key)
                     <div class="field @error('social_links.' . $key) field--error @enderror">
                         <x-hearth-label for="social_links_{{ $key }}" :value="__(':service', ['service' => Str::studly($key)]) . ' ' . __('(optional)')" />
                         <x-hearth-input id="social_links_{{ $key }}" name="social_links[{{ $key }}]"
-                            :value="old('social_links.' . $key, $individual->social_links[$key] ?? '')" />
+                            :value="old('social_links.' . $key, $individual->social_links[$key] ?? '')" hinted="social_links-hint" />
```

Comment:
Should this field have `type="url"`? Same for social media links throughout the application.